NickvisionMoney.GNOME/Views/ShortcutsDialog.cs

```diff
@@ -100,6 +100,12 @@ public ShortcutsDialog(Localizer localizer, Gtk.Window parent)
                                     <property name='accelerator'>&lt;Control&gt;question</property>
                                 </object>
                             </child>
+                            <child>
+                                <object class='GtkShortcutsShortcut'>
+                                    <property name='title'>{localizer["Quit"]}</property>
+                                    <property name='accelerator'>&lt;Control&gt;q</property>
+                                </object>
+                            </child>
```

Comment:
I think it should be below help shortcut

Reply:
Done :+1: